src/encoded/static/components/search.js

```diff
@@ -754,6 +754,22 @@ class Facet extends React.Component {
             // The term exists, but without a key, so don't allow its display.'
             return false;
         });
+
+        // Sort numerical terms by value not by frequency
+        // This should ultimately be accomplished in the back end, but the front end fix is much simpler so we are starting with that
+        if (terms.length > 0) {
+            // For straightforward numerical facets, just sort by value
+            // We have to check the full list for now (until schema change) because some lists contain both numerical and string terms ('Encyclopedia version' under Annotations) and we do not want to sort those by value
+            const numericalTest = (a, b) => !isNaN(a) && !isNaN(b);
```

Comment:
I think you can replace these two lines with `terms.every` unless this is doing something else I don’t see. And in the `every` callback just return whether a single `term.key` value is not NaN. As much as I love `reduce`, it was kind of hard to get my mind around this map/reduce, where `every` would seem more obvious what it’s checking.